README.md

```diff
@@ -1,13 +1,19 @@
-# Project Name
+# Build the game: Guess Who?
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The focus of this week's assignment is about practising JavaScript.
+Create and manipulate objects and arrays. Use different array methods. How to structure the code in functions and plan the game logic.
+To start with, we got a HTML, CSS and JS file provided by Technigo.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The first thing I did was to deep dive into the code that we received from Technigo. To really understand what they have started with.
+Then there were some guidance and suggestion how to approach the project with 6 different steps and one Bonus step.
+I followed the guidance. When the basic logic of the game was working as it should, I continued on with the styling. Added a loading gif and played the game from the user perspective. I noticed some features were not so user friendly, so did try to make some small improvements.
+Added sound effects when flipping the cards and when player win or lose.
+Even though I find the sound effect when flipping the cards a bit annoying, it was a good practise.
```

Comment:
Haha! Yes, it can be a bit annoying with sounds sometime! But as you say, very good practise! 
And fun!!

Reply:
Yeah it was fun for like 1 minute, and the I had to mute it 🙈